doc/changelog/07-vernac-commands-and-options/14740-alizter+split-scheme-vernac.rst

```diff
@@ -0,0 +1,5 @@
+- **Changed:**
+  Scheme Equality is now it's own vernac command, seperating it from other Scheme commands.
+  (`#14740 <https://github.com/coq/coq/pull/14740>`_,
+  fixes `#14735 <https://github.com/coq/coq/issues/14735>`_,
+  by Ali Caglayan).
```

Comment:
Same comment than in some other PRs, IMHO users have no idea what this means to them.

Reply:
If "Equality" used to work in Scheme, then it shouldn't be dropped from scheme_kind without going through the deprecation process.  If it's not dropped then this should be an "Added".

Reply:
I don't think that anything that used to work has been dropped nor that anything as been added though. But I agree with Emilio that "its own vernac command" doesn't mean anything from a user-perspective.

Reply:
Dropped changelog